datahub-web-react/src/app/ingest/source/builder/RecipeForm/RecipeForm.tsx

```diff
@@ -108,6 +116,11 @@ function RecipeForm(props: Props) {
                     {fields.map((field, i) => (
                         <FormField field={field} removeMargin={i === fields.length - 1} />
                     ))}
+                    {type === SNOWFLAKE && (
+                        <ConnectionRequestWrapper>
+                            <ConnectionRequest recipe={displayRecipe} />
```

Comment:
Is this the button? If yes, maybe consider calling it TestConnectionButton

Reply:
correct and sounds good